receiver/splunkhecreceiver/receiver.go

```diff
@@ -231,20 +253,75 @@ func (r *splunkReceiver) Shutdown(context.Context) error {
 	return err
 }
 
-func (r *splunkReceiver) writeSuccessResponse(ctx context.Context, resp http.ResponseWriter, eventCount int) {
+func (r *splunkReceiver) writeSuccessResponseWithAck(ctx context.Context, resp http.ResponseWriter, eventCount int, partitionID string) {
```

Comment:
From the method names I don't understand the difference between `writeSuccessResponseWithAck` and `writeSuccessAckResponse`. I'm probably missing something simple, but can you explain the difference, and when to use each?

Reply:
Ah, the functions were very similarly named unfortunately. But one is writing success response to event sent successful (/services/collector) and the other is writing success response to the query ack endpoint (default /services/collector/ack). They have different response bodies.

Reply:
Okay, that makes more sense. If you can think of a way to make their difference more obvious in the name, that would be great. I could just see it being easy to use the wrong one.

Reply:
I did deleted the method as it is very simple and included its body in the caller.